### DISCUSSION TITLE
Excluding a subdirectory

### DISCUSSION THREAD

**Discussion by dpetka2001** [twpayne/chezmoi#1760] [General]
When i want to add a specific directory from my .config, is there a way to exclude a subdirectory from that specific directory?
For example i want to add my nvim directory but want to exclude nvim/plugged subdirectory from being added to the source state.
Is that possible?

**Comment by twpayne** [comment]
Yes, put the following in your `.chezmoiignore` file:
```
.nvim/plugged
.nvim/plugged/**
```

**Reply by dpetka2001** [reply]
Thank you for the help. I had to also add ".config" in front of that. I think I understand a little better now the terms of target paths and source paths as explained in the reference doc. By default the whole home directory is the destination directory and every file/directory in there can be a potential target if we add it to chezmoi. So the target paths should be relative to the home directory. Thanks again.
